core/src/test/java/com/google/instrumentation/trace/ScopedSpanHandleTest.java

```diff
@@ -28,7 +28,7 @@
 /** Unit tests for {@link ScopedSpanHandle}. */
 @RunWith(JUnit4.class)
 public class ScopedSpanHandleTest {
-  private static final Tracer tracer = Tracer.getTracer();
+  private static final Tracer tracer = Trace.getTracer();
```

Comment:
Suggestion: You could call `Tracer.getNoopTracer()` (or a constructor if it's not the NoopTracer) to ensure that you use the right class.